centaur/src/it/scala/centaur/reporting/BigQueryReporter.scala

```diff
@@ -72,23 +72,28 @@ class BigQueryReporter(override val params: ErrorReporterParams) extends ErrorRe
     }
   }
 
-  override def logCentaurFailure(testEnvironment: TestEnvironment,
-                                 ciEnvironment: CiEnvironment,
-                                 centaurTestException: CentaurTestException)
-                                (implicit executionContext: ExecutionContext): IO[Unit] = {
-    for {
-      callAttemptFailures <- CallAttemptFailure.buildFailures(centaurTestException.metadataJsonOption)
-      jobKeyValueEntries <- params.database.jobKeyValueEntriesIo(centaurTestException.workflowIdOption)
-      metadataEntries <- params.database.metadataEntriesIo(centaurTestException.workflowIdOption)
-      _ <- sendBigQueryFailure(
-        testEnvironment,
-        ciEnvironment,
-        centaurTestException,
-        callAttemptFailures,
-        jobKeyValueEntries,
-        metadataEntries
-      )
-    } yield ()
+  override def logFailure(testEnvironment: TestEnvironment,
+                          ciEnvironment: CiEnvironment,
+                          throwable: Throwable)
+                         (implicit executionContext: ExecutionContext): IO[Unit] = {
+    throwable match {
+      case centaurTestException: CentaurTestException =>
+        for {
+          callAttemptFailures <- CallAttemptFailure.buildFailures(centaurTestException.metadataJsonOption)
+          jobKeyValueEntries <- params.database.jobKeyValueEntriesIo(centaurTestException.workflowIdOption)
+          metadataEntries <- params.database.metadataEntriesIo(centaurTestException.workflowIdOption)
+          _ <- sendBigQueryFailure(
+            testEnvironment,
+            ciEnvironment,
+            centaurTestException,
+            callAttemptFailures,
+            jobKeyValueEntries,
+            metadataEntries
+          )
+        } yield ()
+      case _ =>
+        IO.unit // this ErrorReporter only supports exceptions of CentaurTestException type
```

Comment:
IMHO this behavior should be in the scaladoc for the implementation of this method. If one passed a `Throwable` into a method named `logFailure` I think the expectation would be that it would be logged, not intentionally ignored...